plugins/out_forward/forward.c

```diff
@@ -643,6 +644,39 @@ static int config_set_properties(struct flb_upstream_node *node,
 
     }
 
+    /* compress (implies send_options) */
+    tmp = config_get_property("compress", node, ctx);
+    if (tmp) {
+        if (!strcasecmp(tmp, "")) {
```

Comment:
Replace `""` with `"text"` here, since it is [how Fluentd treat the compress option.](https://github.com/fluent/fluentd/blob/master/lib/fluent/plugin/out_forward.rb#L90)

Note that Fluent Bit does not actually accept an empty string as a parameter
value e.g. the following config is recognized as a string `"\"\""` being passed.

```init
[OUTPUT]
     Name forward
     Compress ""
```